storagemarket/provider.go

```diff
@@ -184,9 +171,30 @@ func (p *Provider) ExecuteDeal(dp *types.ClientDealParams) (pi *api.ProviderDeal
 		return resp.ri, nil
 	}
 
+	log.Infow("scheduled deal for execution", "id", dp.DealUuid)
 	return nil, nil
 }
 
+func (p *Provider) sendDealForAcceptance(ds *types.ProviderDealState) (acceptDealResp, error) {
```

Comment:
nit: suggest renaming to `checkForDealAcceptance`

Reply:
done.